sdk/nodejs/src/index.ts

```diff
@@ -1,13 +1,23 @@
 import { DVCOptions } from './types'
 import { DVCClient } from './client'
+import { DVCCloudClient } from './cloudClient'
 
-export { DVCClient }
+export { DVCClient, DVCCloudClient }
 export * from './types'
 
-export function initialize(environmentKey: string, options?: DVCOptions): DVCClient {
+export type DVCOptionsCloudEnabled = DVCOptions & { enableCloudBucketing: true }
+export type DVCOptionsLocalEnabled = DVCOptions & { enableCloudBucketing: false }
+
+export function initialize(environmentKey: string, options: DVCOptionsLocalEnabled): DVCClient
+export function initialize(environmentKey: string, options: DVCOptionsCloudEnabled): DVCCloudClient
+export function initialize(environmentKey: string, options: DVCOptionsLocalEnabled | DVCOptionsCloudEnabled):
```

Comment:
i think options on this one can just be `DVCOptions` since its a superset of the other two